src/main/kotlin/com/terraformation/backend/accelerator/model/CohortModel.kt

```diff
@@ -0,0 +1,39 @@
+package com.terraformation.backend.accelerator.model
+
+import com.terraformation.backend.db.default_schema.CohortId
+import com.terraformation.backend.db.default_schema.CohortPhase
+import com.terraformation.backend.db.default_schema.ParticipantId
+import com.terraformation.backend.db.default_schema.tables.pojos.CohortsRow
+import com.terraformation.backend.db.default_schema.tables.references.COHORTS
+import org.jooq.Field
+import org.jooq.Record
+
+data class CohortModel<ID : CohortId?>(
+    val id: ID,
+    val name: String,
+    val participantIds: List<ParticipantId>,
```

Comment:
This could maybe be a `Set` instead of a `List` to codify the fact that duplicate IDs aren't allowed.

Also, I wonder if this should be a second type parameter like `ID` so that we can use the type system to forbid setting a list of participant IDs on a `NewCohortModel`. Though at that point it might be cleaner to make `ExistingCohortModel` and `NewCohortModel` entirely separate classes (and leave the ID and participants off the latter) since half the properties would be different between the two.

Reply:
Going with the second type parameter, let me know what you think, I could go either way re: leave as is or entirely separate classes.

Reply:
Scratch that, seeing your comment in #1681, I decided to make it non-nullable and default to an empty set. This allowed me to remove the second type parameter.